lib/repo.ml

```diff
@@ -35,6 +35,17 @@ let local_packages ~recurse ?filter t =
                 "Conflicting definitions for the same package have been found:\n- %s\n- %s"
                 (Fpath.to_string a) (Fpath.to_string b)))
 
+let all_local_packages_names repo =
+  let open Result.O in
+  Bos.OS.Dir.exists repo >>= fun exists ->
+  if not exists then Ok OpamPackage.Name.Set.empty
```

Comment:
This is to be consistent with `local_packages`. Should that be an error instead?